adhocracy4/test/helpers.py

```diff
@@ -33,3 +37,29 @@ def redirect_target(response):
 def render_template(string, context=None):
     context = Context(context or {})
     return Template(string).render(context)
+
+
+@contextmanager
+def skip_background_mail():
```

Comment:
Could you add a comment that briefly summarizes what the aim of this context manager is? I am not sure if I understood it correctly, but if so I would probably write "Re import email tasks module with background_task decorator mocked away. The new implementation calls the function synchronously and checks all arguments for json serilizability".

You probably already checked it, but there is no means provideded by background_tasks to write such tests?

In addition I was also thinking whether we shouldn't make this more generic, so that one can basically give the module as a parameter here.